pkg/strategy/production.go

```diff
@@ -102,6 +102,20 @@ func (c *productionStrategy) Create() []runtime.Object {
 		}
 	}
 
+	if strings.ToLower(c.jaeger.Spec.Storage.Type) == "elasticsearch" {
+		// TODO query and collector will be failing until ES is up - we could wait on readiness probe
+		// TODO we could create ES CR if no es.server-urls are provided (and ES CRD is defined)
+		err := storage.CreateESCerts()
```

Comment:
@jpkrohling @objectiser 

We should decide who and under what conditions the ES instance will be created. 

I would like to create ES CR (assuming es-operator is deployed by OLM and will create ES instance) if:
* flag `es.server-urls`  is not defined 
* there is ES CRD (optional)

so I will create ES here and configure our components to talk to it - mount secrets and use appropriate flags.

Reply:
Sounds reasonable to me, as to use an existing ES cluster they would need to provide the URL.